auacm/app/modules/scoreboard_manager/views.py

```diff
@@ -70,12 +73,13 @@ def getCompetitionData(cid):
 
     for team in team_users.keys():
         team_problems = dict()
-        for problem in comp_problems:
+        for name in comp_problems.keys():
+            problem = comp_problems[name]
             correct = 0
             incorrect = 0
             pointless = 0
             for s in submissions:
-                if not s.pid == problem or not s.username in team_users[team]:
+                if not s.pid == problem['pid'] or not s.username in team_users[team]:
```

Comment:
This is picky, but it is more readable and (I think) more pythonic to say `<somthing> not in <list>` rather than `not <something> in <list>`